docs/scos/dev/upgradability-services/upgradability-reference.md

```diff
@@ -1,19 +1,112 @@
 ---
 title: Upgradability reference
-description: Rerference infomation for evaluator and upgrader tools.
+description: Reference information for evaluator and upgrader tools.
 last_updated: Nov 25, 2021
 template: concept-topic-template
 ---
 
-## Method is overridden with a Factory, Dependency Provider, Repository, or Entity Manager extended
+## Method of an extended class is overridden on the project level
 
-### What is the nature of the upgradability error?
+Factory, Dependency Provider, Repository, and Entity Manager methods belong to the private API. If you extend a core class and override one of its methods, minor releases can cause errors or unexpected changes in functionality.
 
+### Using custom methods on the project level
 
-### Example of code that can cause the upgradability errors
+To avoid the error during updates and achieve the same result, instead of overriding the core methods, introduce custom ones.
```

Comment:
To achieve the same business outcome while being compliant with Spryker customization best practices, instead of overriding the core methods, introduce custom ones.